NitroxPatcher/Patches/CyclopsSonarButton_SonarPing_Patch.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Collections.Generic;
+using System.Linq;
+using Harmony;
+using NitroxClient.GameLogic;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.Core;
+using UnityEngine;
+
+
+namespace NitroxPatcher.Patches
+{
+    class CyclopsSonarButton_SonarPing_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(CyclopsSonarButton);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("SonarPing", BindingFlags.NonPublic | BindingFlags.Instance);
+        public static readonly OpCode JUMP_TARGET_CODE = OpCodes.Ldsfld;
+        public static readonly FieldInfo JUMP_TARGET_FIELD = typeof(SNCameraRoot).GetField("main", BindingFlags.Public | BindingFlags.Static);
+
+        public static void Postfix(CyclopsSonarButton __instance)
```

Comment:
Can we add a description above each patch class that tells what it is doing?

Reply:
I commented the specific methods. I hope that´s what you wanted.